internal/services/nginx/nginx_deployment_data_source.go

```diff
@@ -249,12 +250,14 @@ func (m DeploymentDataSource) Read() sdk.ResourceFunc {
 					output.NginxVersion = pointer.ToString(props.NginxVersion)
 					output.DiagnoseSupportEnabled = pointer.ToBool(props.EnableDiagnosticsSupport)
 
-					if props.Logging != nil && props.Logging.StorageAccount != nil {
-						output.LoggingStorageAccount = []LoggingStorageAccount{
-							{
-								Name:          pointer.ToString(props.Logging.StorageAccount.AccountName),
-								ContainerName: pointer.ToString(props.Logging.StorageAccount.ContainerName),
-							},
+					if !features.FivePointOhBeta() {
```

Comment:
The `logging_storage_account` property in the data source schema needs to be removed conditionally using the `features.FivePointOhBeta` flag as well

Reply:
👍

Reply:
done.